src/commands/project.ts

```diff
@@ -34,10 +33,11 @@ export default class ProjectCommand extends Command {
         },
       ],
     });
-    // TODO: Put message and emojis in configuration
-    const botMessage = await channel.send('Coucou');
-    await botMessage.react('👍');
-    await botMessage.react('👎');
+
+    const botMessage = await channel.send(process.env.DISCORD_PROJECT_MESSAGES);
+    for (const react of process.env.DISCORD_PROJECT_REACT!.split(',')) {
+      await botMessage.react(react)  
+    }
```

Comment:
While I do agree that environment variables are the right tool for accessing the token, I don't think it's particularly suited for this kind of configuration. 

If we want the message to be long enough to have multiple lines, it will become a paint to edit in an environment file. It also becomes obvious that this is not suited because we need to split the reactions by a comma. 

If we use environment variables as configuration though, even temporarily, I think there should at least be two variables for the upvote and downvote reaction instead of just one. 

```suggestion
await botMessage.react(process.env.BOOMBOT_EMOJI_UPVOTE);
await botMessage.react(process.env.BOOMBOT_EMOJI_DOWNVOTE);
```